test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTest.java

```diff
@@ -194,7 +194,7 @@ public void periodicCkpointTrimTest() throws Exception {
         scheduleConcurrently(1, ignored_task_num -> {
             for (int i = 0; i < mapSize; i++) {
                 m2A.put(String.valueOf(i), (long)i);
-                m2B.put(String.valueOf(i), (long)0);
+                m2B.put(String.valueOf(i), (long)i);
```

Comment:
oh, that's the 'wrong' fix, I wanted the maps to differ and for m2B to contains zeros. Anyway, at least you made it consistent.